File: app/src/main/java/com/apps/waziup/data/model/Asset.java
package com.apps.waziup.data.model;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import io.objectbox.annotation.Index;

/**
 * Created by KidusMT on 2/22/2018.
 */

@Entity
public class Asset {

    @Id
    long id;

    @Index
    long asset_id;

    String name;
    String description;
    String image;
    String type;
    String unit;
    String position;
    String latitude;
    String longitude;
    String group;
    String association;
    String note;
    String isPublic;
}

File: app/src/main/java/com/apps/waziup/model/Asset.kt
package com.apps.waziup.model

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id
import java.util.*

/**
 * Immutable POJO that represents a ProjectOld
 * Created by askia on 2/25/2018.
 */

data class Asset(
        @Id var id: Long = 0,
        var type: String? = "Farm",
        var name: String? = null,
        var description: String? = null,
        var position: Int = 0,
        //var location: Location? = null,
        var data: String? = null,
        var date: Date? = null,

        var isPublic: Boolean = false
)